instrumentation/mongo/mongo-3.1/src/test/groovy/MongoClientTest.groovy

```diff
@@ -68,6 +68,27 @@ class MongoClientTest extends MongoBaseTest {
     collectionName = "testCollection"
   }
 
+  def "test create collection with already built ClientSettings"() {
+    setup:
+    def clientOptions = client.mongoClientOptions
+    def newClientOptions = MongoClientOptions.builder(clientOptions).build()
+    MongoDatabase db = new MongoClient(new ServerAddress("localhost", port), newClientOptions).getDatabase(dbName)
+
+    when:
+    db.createCollection(collectionName)
+
+    then:
+    assertTraces(1) {
+      trace(0, 1) {
+        mongoSpan(it, 0, "{\"create\":\"$collectionName\",\"capped\":\"?\"}")
+      }
+    }
+
+    where:
```

Comment:
I think you don't need `where` block here. Just use this values/variable directly in `setup` block

Reply:
using the `where` block like this is consistent with the other tests in this class, so I think it's good